ministryofjustice/laa-review-criminal-legal-aid
# CRIMRE-276-Hide offence class when not provided

## Description of change
Hide offence class when not provided

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMRE-276

## Notes for reviewer
Currently, if an offence has been manually input by a provider on Apply, there will not be an Offence Class set for it. This PR ensures that the offence class caption is only shown if it has been provided.

## Screenshots of changes (if applicable)

### Before changes:
![image](https://user-images.githubusercontent.com/106679087/231815365-f75844eb-2cc7-4c77-b292-9f8228a70ae8.png)

### After changes:
![image](https://user-images.githubusercontent.com/106679087/231815518-9e77c3b8-76be-4f2f-8451-46eee0329617.png)

## How to manually test the feature
- Send an application on Apply that has a manual inputted offence
- View application in Review to check there is no class caption under the offence name
